modules/stackrox-kubernetes-security-platform-support.adoc

```diff
@@ -12,7 +12,7 @@ StackRox supports the StackRox Kubernetes Security Platform versions for up to s
 ====
 For the StackRox Kubernetes Security Platform, StackRox supports:
 
-* The latest released version (reffered as N).
+* The latest released version (referred to as "N").
```

Comment:
The "Letters as letters" section in the ISG makes me think this maybe should be _N_, but I don't think it's a showstopper.

Reply:
I agree with your suggestion and made the change to _N_, @kalexand-rh - thanks!